src/verify/mod.rs

```diff
@@ -27,6 +27,15 @@ pub trait Credential {
     fn parse_claims(claims: ClaimsSet) -> Result<HashMap<String, ClaimValue>>;
 }
 
+pub fn retrieve_entry_from_status_list(status_list: String, idx: usize) -> Result<u8, DecodeError> {
+    let status_list: JsonStatusList =
+        serde_json::from_str(status_list.as_str())
+            .map_err(|e| DecodeError::Zlib(e.into()))?;
+    let bitstring = status_list
+        .decode(None)?;
+    bitstring.get(idx).ok_or(DecodeError::MissingStatusList)
```

Comment:
As per Jacob's comment above, if you are using anyhow, can you use that for this error too (since `MissingStatusList` is different than the index not being in the status list)